build/tasks/task-serve.js

```diff
@@ -25,8 +26,14 @@ gulp.task('serve', async () => {
     '--port', ports[0],
   ];
 
+  const runnerArgs = [
+    ...args,
+    ...(yargs.argv.browser ? ['--browser', yargs.argv.browser] : []),
```

Comment:
Oh also, maybe this should be renamed to `dummy-browser` or similar to be clear that it's only applicable for that platform?